rules/windows/builtin/security/win_security_disable_event_auditing_critical.yml

```
@@ -0,0 +1,49 @@
+title: Important Windows Event Auditing Disabled
+id: ab4561b1-6c7e-48a7-ad08-087cfb9ce8f1
+related:
+    - id: 69aeb277-f15f-4d2d-b32a-55e883609563
+      type: derived
+status: test
```

New one should be experimental
```suggestion
status: experimental
```

---

Its not new, its just a derived version of an already existing rule that was separated for clarity.